### DISCUSSION TITLE
如何配置http代理

### DISCUSSION THREAD

**Discussion by lgcshy** [swuecho/chat#227] [Q&A] [answered]
qiang太厚，需要走v 2 r a y搭建的http代理

**Comment by swuecho** [comment]
1. 用旁路由模式 
2. 全局代理
3. 我这边提供带配置支持 但是我不太懂 需要有懂的人帮忙

**Comment by swuecho** [comment]
https://github.com/swuecho/chat/issues/166

如果你有个proxy url 我可以把这个加上试试.

v2ray 貌似你让 openai.com 走代理就行了.

**Comment by swuecho** [comment]
https://github.com/swuecho/chat/pull/232

不知道是否对你有用

如果你弄成功了, 可以写一下过程,帮助后来人.

**Comment by swuecho** [comment, marked as answer]
https://github.com/Chanzhaoyu/chatgpt-web/issues/1593#issuecomment-1521168205

```
你没有使用http/socks5代理，一般是不成功的。 clash的本地代理地址默认是http://127.0.0.1:7890。

你可以加上：  --env HTTPS_PROXY=http://127.0.0.1:7890 试试看。
```

OPENAI_PROXY_URL=http://127.0.0.1:7890  应该可以

**Reply by swuecho** [reply]
最新 0.8.16 应该可以了
